admin/client/src/legacy/ModelAdmin.js

```diff
@@ -3,6 +3,8 @@
  */
```

Comment:
I can't see how this file is actually getting included, and my suspicion is that the code is so edge-case that tests don't file if we remove it entire.

Can we see some evidence that the `.cms-content-tools #Form_SearchForm` entwine rule has been bundled?

If not, do we need to include these files in the bundle-legacy entrypoint?